bgruening.yaml

```diff
@@ -195,3 +195,7 @@ tools:
   - name: add_line_to_file
     owner: bgruening
     tool_panel_section_label: Text Manipulation
+
+  - name: mztab2tsv
+    owner: bgruening
```

Comment:
you can add this to the galaxyp file with section Proteomics I think or `Convert Formats`